.github/workflows/verify-repo-tags.yml

```diff
@@ -0,0 +1,80 @@
+name: "Verify Repo Tags"
+
+on:
+  schedule:
+    - cron:  '0 7 * * *'
+  workflow_dispatch: {}
+
+jobs:
+  configure-github:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Verify Repo Tags
+        id: verify_repo_tags
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: |
+          output=$(bundle exec rake github:verify_repo_tags)
+
+          untagged_repos=$(echo "$output" | grep 'Untagged govuk repos:' -A 1 | tail -n +2)
+          falsely_tagged_repos=$(echo "$output" | grep 'Falsely tagged govuk repos:' -A 1 | tail -n +2)
+
+          echo "::set-output name=UNTAGGED_REPOS::$untagged_repos"
+          echo "::set-output name=FALSELY_TAGGED_REPOS::$falsely_tagged_repos"
+
+      - name: Notify failure
+        uses: slackapi/slack-github-action@v1
+        if: ${{ failure() }}
+        with:
+          payload: |
+            {
+              "text": "Verify Repo Tags: Unresolved issues found. Check the build logs for details.",
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "Verify Repo Tags: Unresolved issues found. Check the build logs for details."
+                  }
+                },
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "Untagged govuk repos:\n${{ steps.verify_repo_tags.outputs.UNTAGGED_REPOS }}"
+                  }
+                },
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "Falsely tagged govuk repos:\n${{ steps.verify_repo_tags.outputs.FALSELY_TAGGED_REPOS }}"
+                  }
+                },
+                {
+                  "type": "actions",
+                  "elements": [
+                    {
+                      "type": "button",
+                      "text": {
+                        "type": "plain_text",
+                        "text": "View Workflow"
+                      },
+                      "url": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}",
+                      "style": "primary"
+                    }
+                  ]
```

Comment:
Similarly, we have a [working method for buttons in govuk-user-reviewer](https://github.com/alphagov/govuk-user-reviewer/blob/39865181c8c95d54b9c4e91b7e120e3886429674/.github/workflows/review-users.yml#L49-L57), if you wanted a less risky merge.